processor/src/operations/mod.rs

```diff
@@ -139,22 +149,22 @@ impl Process {
             Operation::AdvPop => self.op_advpop(host)?,
             Operation::AdvPopW => self.op_advpopw(host)?,
 
-            Operation::MLoadW => self.op_mloadw()?,
-            Operation::MStoreW => self.op_mstorew()?,
+            Operation::MLoadW => self.op_mloadw(error_ctx)?,
+            Operation::MStoreW => self.op_mstorew(error_ctx)?,
```

Comment:
We propagate `error_ctx` to some operations but not others - is this because we don't need it in other places, or we just haven't gotten to the other ones yet?

Reply:
That's right - I think in the end, most (all?) `op_*` methods that can fail will need to `error_ctx` to be able to construct a nice error message.